### PR TITLE
Adding popup for feedback and removing unused GA_ID from static files

### PR DESCRIPTION
**Pull request by teochengyong**
- Added feedback form with formSG 
feedback
image
- Added analytic events to track the submission rate

### CODE REVIEW

File: _includes/feedback.html [hunk 1]
```diff
@@ -25,6 +26,79 @@
...
+          <p>
+            If you've got a moment, please tell us what we did right so that we
+            can do more of it.
+          </p>
+        </section>
+        <footer class="sgds-modal-card-foot is-flex-justify-fs">
+          <a
+            class="sgds-button is-outlined is-primary js-send-feedback-modal"
+            target="_blank"
+            href="{{ site.feedback_form }}?60efdac78c8a4a0011f25c69={{ page.url | uri_escape }}"
```
Comment (tankunsheng) [inline _includes/feedback.html:56]: use {{ site.feedback_form }}?{{ site.feedback_form_url_field }}={{ page.url | uri_escape }} as well?

File: _includes/feedback.html [hunk 2]
```diff
@@ -36,5 +110,16 @@
       </span>
       <span class="has-text-success">Sent. Thank you for the feedback!</span>
     </div>
+    
+    <div class="col is-12">
+      <a
+        target="_blank"
+        href="{{ site.feedback_form }}?60efdac78c8a4a0011f25c69={{ page.url | uri_escape }}"
```
Comment (tankunsheng) [inline _includes/feedback.html:117]: use {{ site.feedback_form }}?{{ site.feedback_form_url_field }}={{ page.url | uri_escape }} as well?

File: _includes/feedback.html [hunk 3]
```diff
@@ -25,6 +26,79 @@
...
+      id="feedback-modal-useful-yes"
+      aria-labelledby="developer portal feedback"
+      role="dialog"
+    >
+      <div class="sgds-modal-background"></div>
+
+      <div class="sgds-modal-card">
+        <header class="sgds-modal-card-head">
+          <p class="sgds-modal-card-title" id="sgds-modal-title">
+            Did this page help you? Yes
```
Comment (tankunsheng) [inline _includes/feedback.html:41]: minor standardization 
"Did this page help you? **-** Yes"

File: _includes/feedback.html (hunk 1 above)
Comment (teochengyong) [reply _includes/feedback.html]: Yup will replace this

File: _includes/feedback.html (hunk 2 above)
Comment (teochengyong) [reply _includes/feedback.html]: yup good catch 👍

**Review by tankunsheng — APPROVED** [review]

**Pull request merged**